dominion/frontend/src/component/Footer/index.js

```diff
@@ -1,18 +1,188 @@
 import React from 'react';
+import PropTypes from 'prop-types';
+
 import { Grid, Typography } from '@material-ui/core';
+import { withStyles } from '@material-ui/core/styles';
+
+import SocialMedia from '../SocialMedia';
+
+import background from '../../assets/images/bg/background.png';
+import Partners from './Partners';
+
+const styles = theme => ({
+  root: {
+    flexGrow: 1,
+    padding: theme.spacing.unit * 3,
+    backgroundImage: `url(${background})`,
+    backgroundRepeat: 'no-repeat',
+    backgroundSize: 'cover',
+    [theme.breakpoints.up('md')]: {
+      height: '50vh'
```

Comment:
`v` units means screen size. Here it means the footer needs to be 50% of the screen height. Are you sure you want a footer that high?